website_demo/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
```

Comment:
What does this file do? I think `mstile-150x150.png` is not relevant to our website. Please consider removing or updating